### ISSUE TITLE
the ability to add entire Surah

### ISSUE THREAD

**Issue by medo2132112** [AmmarCodes/obsidian-quran-helper-plugin#3]
first of all جزاك الله خير 
amazing work but as a user i think it will be better to add entire Surah because sometimes i use obsidian in my tablet add read Surah Al-Kahf especially on Friday's with a help form [Obsidian Teleprompter](https://github.com/lumetrium/obsidian-teleprompter) 
![SmartSelect_٢٠٢٤٠٧٠٨_١٤٥٠٢٨_Obsidian](https://github.com/AmmarCodes/obsidian-quran-helper-plugin/assets/172226789/118341c7-e796-45ae-9fc5-395d9f1bed05)

**Comment by AmmarCodes** [comment]
وإياكم!

I'm considering adding an option for the whole surah. Just wondering, is it easy to read the full surah in one page?

**Comment by medo2132112** [comment]
Certainly, with the implementation of Obsidian Teleprompter, I believe it is feasible

**Comment by muhamed-saleh** [comment]
I am recently using obsidian to help with memorizing the quran
and inserting a whole surah is a feature I am looking for

**Comment by AmmarCodes** [comment]
@muhamed-saleh @medo0o0o0 this feature has been add in the [latest version](https://github.com/AmmarCodes/obsidian-quran-helper-plugin/releases/tag/1.4.3), please try it out.
